packages/pigeon/lib/dart_generator.dart

```diff
@@ -236,20 +236,21 @@ void _writeFlutterApi(
       );
       indent.writeln('$returnType ${func.name}($argSignature);');
     }
-    indent.write('static void setup(${api.name}$nullTag api) ');
+    indent.write(
+        'static void setup(${api.name}$nullTag api, {BinaryMessenger$nullTag binaryMessenger}) ');
```

Comment:
Off-topic, but the change here reminded me that I wanted to raise this: could we fix the fact that this name is wrong (it should be a verb, `setUp`) in all of the generators? If we don't want a breaking change we could have `setup` as a deprecated passthrough that calls `setUp`.

Reply:
Yea that sounds reasonable, we have talked about this in the past.  I filed an issue for it: https://github.com/flutter/flutter/issues/93062